docs/snippets/final-flow-state.mdx

```diff
@@ -0,0 +1,34 @@
+The final state of a flow is determined by its **_return value_**. The following rules apply:
+
+- If an exception is raised directly in the flow function, the flow run is marked as `FAILED`.
+- If a flow returns a manually created state, it is used as the state of the final flow run. This allows for manual determination of final state.
+- If a flow returns an iterable of states, the presence of _any_ `FAILED` state will cause the run to be marked as `FAILED`.
+
+In _any other situation_ in which the flow returns without error, it will be marked as `COMPLETED`. 
+
+<Warning>
+If you manipulate states programmatically, you can create situations in which tasks 
+within a flow can fail and not cause flow run failure. 
+For example:
+```python
+from prefect import flow, task 
+
+
+@task 
+def add_one(x):
+    return x + 1
+
+
+@flow 
+def my_flow():
+    # avoided the raising of an exception with return_state flag
```

Comment:
as is, runs off the page
```suggestion
    # avoided raising an exception via `return_state=True`
```
## before
![image](https://github.com/user-attachments/assets/fece484d-ff68-4cec-93cb-1d155ee2ee89)

## after
![image](https://github.com/user-attachments/assets/bd46ab0f-eae3-43ed-95a0-bd1430ba5327)